pkg/models/model_performer.go

```diff
@@ -298,15 +308,6 @@ func (p *Performer) CopyFromCreateInput(input PerformerCreateInput) error {
 func (p *Performer) CopyFromUpdateInput(input PerformerUpdateInput) error {
 	CopyFull(p, input)
 
-	if input.Image != nil {
-		var err error
-		p.Image, err = p.TranslateImageData(input.Image)
-
-		if err != nil {
-			return err
-		}
-	}
-
```

Comment:
This looks unrelated to the uuid change. Why was this removed from here?

Reply:
My bad, fixed.